extensions/widgets/browser/browser.lcb

```diff
@@ -24,14 +24,14 @@ Licensed under the terms of the Creative Commons Attribution 3.0 Unported Licens
 /*
 This widget displays web content within a native web browser view.
 
-Name: url
+Name: URL
 Type: property
 
 Syntax:
-set the url of <widget> to <pUrl>
-get the url of <widget>
+set the URL of <widget> to <pUrl>
```

Comment:
Why are these uppercased? Remember that properties are only accessible from LCS, and there property names are camel-cased.